modules/administration-guide/pages/removing-user-data-in-compliance-with-the-gdpr.adoc

```diff
@@ -7,34 +7,29 @@
 [id="removing-user-data-in-compliance-with-the-gdpr"]
 = Removing user data in compliance with the GDPR
 
-You can remove a user’s data in compliance with the link:https://gdpr.eu/[General Data Protection Regulation (GDPR)] that enforces the right of individuals to have their personal data erased.
+You can remove a user’s data on {ocp} in compliance with the link:https://gdpr.eu/[General Data Protection Regulation (GDPR)] that enforces the right of individuals to have their personal data erased. The process for other {kubernetes} infrastructures might vary. Follow the user management best practices of the provider you are using for the {prod} installation.
 
 WARNING: Removing user data as follows is irreversible! All removed data is deleted and unrecoverable!
 
 .Prerequisites
 
-* An active session with administrative permissions for {prod-short}. See xref:configuring-the-administrative-user.adoc[].
-
-* An active `oc` session with administrative permissions for the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+* An active `oc` session with administrative permissions for the {ocp} cluster. See link:https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
```

Comment:
```suggestion
* An active `oc` session with administrative permissions for the {ocp} cluster. See link:https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
* Verify that the resources associated with the selected user (such as projects, roles, or service accounts) were deleted.
```

Reply:
this should be part of procedure, not pre-req

Reply:
well, this step can vary, so I would leave it as was done originally

Reply:
I came here from the forum post.

@ibuziuk and @deerskindoll , fyi, it's possible to make this a step in the procedure even if it doesn't apply to all users by adding `Optional: ` at the front, for example:
```
. Optional: Verify that the resources associated with the selected user (such as projects, roles, or service accounts) were deleted.
 ```
or
```
. Optional: Delete the resources associated with the selected user (such as projects, roles, or service accounts).
 ```